Sources/PackageGraph/PinsStore.swift

```diff
@@ -117,7 +110,7 @@ public final class PinsStore {
     public func createConstraints() -> [RepositoryPackageConstraint] {
```

Comment:
you can remove this method, it's dead code

Reply:
Ah right, I'll make a follow up to clean this up.

Reply:
#2793